### PR TITLE
[PIE-2215] Fix spelling and punctuation errors

### PR DESCRIPTION
**Pull request by sgregglives**
## PR description
Fix spelling and punctuation errors
## Fixed Issue(s)

PIE-2215

### CODE REVIEW

File: docs/HowTo/Quickstart.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 
-## Prequisites
+## Prerequisites
 
 [Orion](../HowTo/Install-Binaries.md)
 
 ## Starting Orion and Sending a Payload
 
-### 1. Create Password File
+### Create Password File
```
Comment (MadelineMurray) [inline docs/HowTo/Quickstart.md]: Create password file 

Needs to be sentence case not title case. Applies to following sections also.

File: docs/HowTo/Quickstart.md [hunk 2]
```diff
@@ -36,19 +36,19 @@ clienturl = "http://127.0.0.1:8888/"
 clientport = 8888
 publickeys = ["nodeKey.pub"]
 privatekeys = ["nodeKey.key"]
-passwords = "/<pathToPasswordFile>"/passwordFile
+passwords = "<PathToPasswordFile>"
```
Comment (MadelineMurray) [inline docs/HowTo/Quickstart.md]: This has dropped the actual filename - ie, the updated version indicates this property is a path only which isn't correct

File: docs/HowTo/Quickstart.md (hunk 1 above)
Comment (sgregglives) [reply docs/HowTo/Quickstart.md]: This is being fixed by https://github.com/PegaSysEng/doc.orion/pull/20

File: docs/HowTo/Quickstart.md (hunk 2 above)
Comment (sgregglives) [reply docs/HowTo/Quickstart.md]: Updated.

**Review by MadelineMurray — APPROVED** [review]

**Pull request merged**